tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@tsconfig/create-react-app/tsconfig.json",
   "compilerOptions": {
+    "jsx": "react-jsx",
```

Comment:
Do you have a docs link for what this does? Things seemed to be working ok for me in locally dev with VSCode

Reply:
My VSC would not compile/lint without it
https://www.typescriptlang.org/tsconfig#jsx
https://stackoverflow.com/a/65539274/5093905

Reply:
hrm, I wonder it worked for me 🤷